bt buổi 10/main.js

```diff
@@ -0,0 +1,96 @@
+function getMultiplication() {
+    for(let i = 0; i < 10; i++) {
+        console.log(6 * i)
+    }
```

Comment:
indent 2 dấu cách, 1 tab = 2 dấu cách
Đọc: https://viblo.asia/p/javascript-style-guide-and-coding-conventions-924lJqVmZPM